packages/expo-app-integrity/src/AppIntegrity.ts

```diff
@@ -74,3 +74,44 @@ export async function requestIntegrityCheck(requestHash: string) {
   }
   return ExpoAppIntegrity.requestIntegrityCheck(requestHash);
 }
+
+/**
+ * Checks if hardware attestation is supported on this device.
+ * @return A Promise that is fulfilled with a boolean indicating support.
+ * @platform android
+ */
+export async function isHardwareAttestationSupported() {
+  if (Platform.OS !== 'android') {
+    throw new Error('isHardwareAttestationSupported is only available on Android');
+  }
+  return ExpoAppIntegrity.isHardwareAttestationSupported();
+}
```

Comment:
How about if we just return a boolean instead of throwing here? Users won't need try/catch then
```suggestion
export async function isHardwareAttestationSupported() {
  if (Platform.OS !== 'android') {
    return false
  }
  return ExpoAppIntegrity.isHardwareAttestationSupported();
}
```